File: _retired/stories/examples/ts/manage-customer-orders/place-new-order/enter-shipping-address/enter-shipping-address-stories.ts
import type { Story } from '../../story-types'

export const EnterShippingAddress = {
  story:       'Enter Shipping Address',
  actor:       'Customer',
  domainTerms: ['Shipping Address', 'Address Line', 'Postal Code', 'Country'],
  evidence:    [],

} as const satisfies Story

File: practices/stories/examples/manage-customer-orders/py/manage-customer-orders/place-new-order/enter-shipping-address/enter_shipping_address_stories.py
"""Story data - regeneratable. Do not add logic or imports."""

from __future__ import annotations

from typing import Final


ENTER_SHIPPING_ADDRESS: Final = {
    "story":        "Enter Shipping Address",
    "actor":        "Customer",
    "domain_terms": ("Shipping Address", "Address Line", "Postal Code", "Country"),
    "evidence":     (),
}
